### ISSUE TITLE
Using CLion+Rust debugger cannot find "libui.so.0"

### ISSUE THREAD

**Issue by mitchhentges** [rust-native-ui/libui-rs#72]
I've set up CLion with the [:tada: IntelliJ rust plugin :tada:](https://intellij-rust.github.io/), but I'm having issues running the debugger:

> `error while loading shared libraries: libui.so.0: cannot open shared object file: No such file or directory`

I located `libui.so.0`:
```
$ find . -name 'libui.so.0'          
./target/debug/build/ui-sys-f439c2f1b0cb0138/out/build/out/libui.so.0
```

So, I tried setting `LD_LIBRARY_PATH` before running CLion:
```
$ LD_LIBRARY_PATH=/home/mitch/dev/pult/target/debug/build/ui-sys-f439c2f1b0cb0138/out/build/out snap run clion
```

However, I'm still getting "error while loading shared libraries" :thinking: 

I'm not super familiar with how shared libraries work, so I highly doubt that this issue is specific to `libui-rs` - there's just a chance that a `libui-rs`-er has run into this issue, too :smile: :man_shrugging:

**Comment by LuciferK9** [comment]
There's an option to set environment variables in the "run configurations". You can try setting the environment variable there.

**Comment by mitchhentges** [comment]
Good idea, though that has the same result as exporting the variable before running clion: it can't find the shared library

**Comment by frederickjjoubert** [comment]
I'm working on a rust project using CLion and the Rust plugin, my project uses the Bevy game engine and I'm getting the same type of error:

`error while loading shared libraries: libbevy_dylib-3888e9814c2a97bf.so: cannot open shared object file: No such file or directory`

Were you able to find a solution?
